Runtime/codebase/SolanaWalletAdapterWebGL/SolanaWalletAdapterWebGL.cs

```diff
@@ -237,15 +235,13 @@ public static void OnAllTransactionsSigned(string signatures)
                 return;
             }
             string[] signaturesList = signatures.Split(',');
```

Comment:
Since the wallet adapter spec changed and it's now returning the full transactions I would rename the fields accordingly, both `signatures` and `signaturesList`

Reply:
I've updated the parameter names, as requested. The fields now reflect that the full transactions are returned instead of just signatures. Let me know if further adjustments are needed.